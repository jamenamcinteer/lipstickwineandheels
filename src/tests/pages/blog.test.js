import React from 'react'
import { shallow } from 'enzyme'
import BlogIndex from '../../pages/blog'
import data from '../fixtures/data'

test('should render blog page correctly', () => {
  const wrapper = shallow(<BlogIndex data={data.data} />)
  expect(wrapper).toMatchSnapshot()
})
