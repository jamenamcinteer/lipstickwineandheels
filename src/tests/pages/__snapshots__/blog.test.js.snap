// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render blog page correctly 1`] = `
<PageTransition>
  <div
    style={
      Object {
        "background": "#fff",
      }
    }
  >
    <HelmetWrapper
      defer={true}
      encodeSpecialCharacters={true}
    >
      <html
        lang="en"
      />
      <title>
        Blog | 
        Lipstick, Wine, and Heels
      </title>
      <meta
        content="Lipstick, Wine, and Heels is a lifestyle blog dedicated to chic living and nerdy topics. I am a woman in tech who appreciates fashion, fine food and drink, crafty hobbies, fitness, and cats."
        name="description"
      />
    </HelmetWrapper>
    <div
      className="heroContainer"
    >
      <div
        className="hero"
      >
        <h1>
          Blog
        </h1>
      </div>
    </div>
    <div
      className="wrapper"
    >
      <ul
        className="article-list"
      >
        <li
          key="this-is-a-sample-blog-post"
        >
          <Component
            article={
              Object {
                "date": "August 30, 2018",
                "slug": "this-is-a-sample-blog-post",
                "tags": Array [
                  Object {
                    "slug": "tech",
                    "title": "Tech",
                  },
                ],
                "title": "This is a sample blog post",
              }
            }
          />
        </li>
        <li
          key="hello-world"
        >
          <Component
            article={
              Object {
                "date": "August 28, 2018",
                "slug": "hello-world",
                "tags": Array [
                  Object {
                    "slug": "tech",
                    "title": "Tech",
                  },
                  Object {
                    "slug": "travel",
                    "title": "Travel",
                  },
                ],
                "title": "Hello World!",
              }
            }
          />
        </li>
      </ul>
    </div>
  </div>
</PageTransition>
`;
